app/.prettierrc

```
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "none",
+  "singleQuote": true,
+  "printWidth": 120
```

```suggestion
  "printWidth": 100
```
The docs suggest a line length / printWidth of 80, but we could split the difference here and go for 100. https://prettier.io/docs/en/options.html#print-width